RELEASE_NOTES.md

```diff
@@ -18,17 +18,12 @@
 This document contains changes of oneTBB compared to the last release.
 
 ## Table of Contents <!-- omit in toc -->
-- [New Features](#new_features)
 - [Known Limitations](#known-limitations)
 - [Fixed Issues](#fixed-issues)
 - [Open-source Contributions Integrated](#open-source-contributions-integrated)
 
-## :white_check_mark: New Features
-- Improved support and use of the latest C++ standards for parallel_sort that allows using this algorithm with user-defined and standard library-defined objects with modern semantics.
-- The following features are now fully functional: task_arena extensions, collaborative_call_once, adaptive mutexes, heterogeneous overloads for concurrent_hash_map, and task_scheduler_handle.
-- Added support for Windows* Server 2022 and Python 3.10.
-
 ## :rotating_light: Known Limitations
+- A static assert will cause compilation failures in oneTBB headers when compiling with clang 12.0.0 or newer if using the LLVM standard library with -ffreestanding and C++11/14 compiler options.
```

Comment:
```suggestion
- A static assert causes compilation failures in oneTBB headers when compiling with Clang* 12.0.0 or newer. The reason is using the LLVM* standard library with -ffreestanding and C++11/14 compiler options.
```

Reply:
@alexandraepan the compilation failure is only seen if LLVM 12 or newer is used and the -ffreestanding option is used along with -std=c++11 or -std=c++14. The proposed wording reads as though the assert fails in all cases where Clang 12.0.0 or newer is used.

Reply:
@timmiesmith okay, thanks for the clarification

Reply:
Updated the comment